Frontend/items.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE HTML>
+<html lang='en'>
+<head>
+    <title>Welcome to Lean Fridge</title>
+
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css' integrity='sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7' crossorigin='anonymous'>
+
+<script type = 'text/javascript'>
+    var stock_array = [
+            {item_id: 1, Name: 'apple', count: '5', purchased_on: '2016-01-22', detail: 'organic Fuji apples'},
+            {item_id: 2, Name: 'orange', count: '4', purchased_on: '2016-01-21', detail: 'Navel oranges for smoothie'},
+            {item_id: 3, Name: 'cucumber', count: '6', purchased_on: '2016-01-24', detail: 'cool as a cucumber'},
+            {item_id: 4, Name: 'almond milk', count: '1', purchased_on: '2016-01-20', detail: 'For breakfast'},
+            {item_id: 5, Name: 'Quinoa', count: '1', purchased_on: '2016-01-24', detail:'1 bag, 1.5 pound'},
+            {item_id: 6, Name: 'potato', count: '6', purchased_on: '2016-01-23', detail:'purple and red potatoes'},
+            {item_id: 7, Name: 'chocolate', count: '2', purchased_on: '2016-01-19', detail:'dark chocolate, good for hot drink'}
+            ]
+
+        console.log(stock_array.length);
+
+
+    function createTable(){
+
+        var table_ref = document.getElementById('table_body');
+
+        for(var i = 0; i < stock_array.length; i++){
```

Comment:
this loop has a LOT of repeated code. can u think of ways to simplify and make it more concise?
this is one option:
```
column_names   =  { 0 : "item_id", 1: "Name", 2 .... }
function createCell(row, row_index, column_index, column_name) {
  var cell = row.insertCell(column_index); 
  var text = document.createTextNode(stock_array[row_index][column_name]);
  cell.appendChild(text);
  return cell;
}
```

and then call this from here in a loop:
```
row = table_ref.insertRow(0);
for (var j = 0; j <= 3; j++) {
  var cell = createCell(row, i, j, column_names[j]);
  row.appendChild(cell);
}
```


can write something similar for the last 3 columns which are not text but buttons. think about it!